applications/webhooks/src/modules/webhooks_notifications.erl

```diff
@@ -0,0 +1,149 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2017, 2600Hz INC
+%%%
+%%% @contributors
+%%%-------------------------------------------------------------------
+-module(webhooks_notifications).
+
+-export([init/0
+        ,bindings_and_responders/0
+        ,account_bindings/1
+        ,handle_event/2
+        ]).
+
+-include("webhooks.hrl").
+
+-define(ID, kz_term:to_binary(?MODULE)).
+-define(HOOK_NAME, <<"notifications">>).
+-define(NAME, <<"Notifications Webhook">>).
+-define(DESC, <<"Fire a webhook when a notification event is triggered in Kazoo">>).
+
+-define(TYPE_ALL
+       ,kz_json:from_list([{<<"friendly_name">>, <<"All">>}
+                          ,{<<"description">>, <<"This event is triggered for any notification events">>}
+                          ])
+       ).
+
+-define(ALL_TYPE_MODIFIERS(Modifiers), kz_json:from_list([{<<"all">>, ?TYPE_ALL} | Modifiers])).
+
+-define(TYPE_MODIFIERS(Modifiers)
+       ,kz_json:from_list([{<<"type">>, <<"object">>}
+                          ,{<<"items">>, kz_json:merge(?ALL_TYPE_MODIFIERS(Modifiers), kz_json:new())} %% merge to sort :)
+                          ])
+       ).
+
+-define(METADATA(Modifiers)
+       ,kz_json:from_list([{<<"_id">>, ?ID}
+                          ,{<<"name">>, ?NAME}
+                          ,{<<"description">>, ?DESC}
+                          ,{<<"modifiers">>
+                           ,kz_json:from_list([{<<"type">>, ?TYPE_MODIFIERS(Modifiers)}])
+                           }
+                          ])
+       ).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec init() -> 'ok'.
+init() ->
+    init(get_notifications_definition(), []).
+
+-spec init(kapi_definition:apis(), kz_json:objects()) -> 'ok'.
+init([], Acc) ->
+    _ = webhooks_util:init_metadata(?ID, ?METADATA(Acc)),
+    'ok';
+init([EventDefinition|Rest], Acc) ->
+    Description = kz_json:from_list([{<<"friendly_name">>, kapi_definition:friendly_name(EventDefinition)}
+                                    ,{<<"description">>, kapi_definition:description(EventDefinition)}
+                                    ]
+                                   ),
+    init(Rest, [{kapi_definition:name(EventDefinition), Description} | Acc]).
```

Comment:
The spec says the second arg is `kz_json:objects()` but this looks like `[{ne_binary(), kz_json:object()}]`

Reply:
fixed